tests/test_server.py

```
@@ -123,10 +123,11 @@ async def test_get_nodes(server, mock_proxmox):
     response = await server.mcp.call_tool("get_nodes", {})
 
     # The response should be formatted text, not JSON
-    assert len(response) == 1
-    assert "Proxmox Nodes" in response[0].text
-    assert "node1" in response[0].text
-    assert "node2" in response[0].text
+    assert len(response) == 1, f"Expected single response item, got {len(response)}"
+    response_text = response[0].text
+    assert "Proxmox Nodes" in response_text, f"Expected 'Proxmox Nodes' header not found in: {response_text[:200]}"
```

Ruff (0.11.9)

Line too long (115 > 100)

(E501)